packages/node_modules/@ciscospark/react-redux-spark/src/component.js

```diff
@@ -22,39 +23,59 @@ const injectedPropTypes = {
 const propTypes = {
   accessToken: PropTypes.string,
   guestToken: PropTypes.string,
+  sdkInstance: PropTypes.object,
   ...injectedPropTypes
 };
 
 const defaultProps = {
   accessToken: '',
-  guestToken: ''
+  guestToken: '',
+  sdkInstance: undefined
 };
 
-const PLUGINS = ['authorization', 'logger', 'people', 'phone', 'rooms'];
+const PLUGINS = ['authorization', 'logger', 'meetings', 'people', 'phone', 'rooms'];
 const INTERNAL_PLUGINS = ['conversation', 'feature', 'flag', 'mercury', 'presence', 'search', 'team'];
 
 export class SparkComponent extends Component {
   constructor(props) {
     super(props);
 
-    this.loadSparkInstance = this.loadSparkInstance.bind(this);
+    this.storeSDKInstance = this.storeSDKInstance.bind(this);
   }
 
   componentDidMount() {
     const {
       accessToken,
       guestToken,
-      spark
+      spark,
+      sdkInstance
     } = this.props;
 
+    // Get the sdk instance from redux state
```

Comment:
Like the added comments to help improve readability.

Reply:
"Commenting code, not just for demos anymore!" -Weeks 2020